tests/suites/test_suite_ssl.function

```diff
@@ -743,6 +743,16 @@ void test_callback_buffer_sanity()
     TEST_ASSERT( mbedtls_test_buffer_put( &buf, NULL, 0 ) == 0 );
```

Comment:
Commit message "Add a test for mbedtls_test_buffer that fails and have to be fixed" should say "Add a test for mbedtls_test_buffer that fails and has to be fixed"

Also, it would be good to expand the commit message to explain what is failing, how it fails, what the fix is, and why the fix is a good one.

Reply:
I changed it and added information about this error to commit message